dist/main.js

```diff
@@ -0,0 +1,45 @@
+class Team {
+
+    renderPlayers(data) {
```

Comment:
You should separate file and concepts, especially when using MVC, write your render functions in a different place